### PR TITLE
Add codeclimate_circle_ci_coverage gem

### PR DESCRIPTION
**Pull request by rdunlop**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# run coverage when on CI
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start 'rails' do
+    add_filter '/spec/'
```
Comment (jhanggi) [inline spec/spec_helper.rb]: Should this also run over `vendor/engines/**/spec/`? Or does this filter already hit it?

Comment (rdunlop) [inline spec/spec_helper.rb]: I don't know.

**Comment by benlinton** [comment]
Nice addition!

`CI=true bundle exec rspec`
`open ./coverage/index.html`

We're at ~34% coverage.

**Comment by jhanggi** [comment]
I got 84% when I ran it.

Looks like it is filtering the engines spec directories, so I think we're good there. I think we should probably `.gitignore` the coverage directory, but other than that :+1:

**Pull request merged**